README.md

```diff
@@ -15,3 +15,40 @@ To run the site locally with Docker, use the following command:
 ```bash
 docker run --rm --label=jekyll --volume=$(pwd):/srv/jekyll  -it -p 127.0.0.1:4000:4000 jekyll/jekyll jekyll serve
 ```
+
+
+
+#### Side note for those on non-linux machines
+ 
```

Comment:
Don't *need* it per-se; Docker machine would work. 

Minikube + xhyve automatically sets up the shared folders properly (i.e., under /Users/<username>) and configures the permissions for making changes locally and syncing them to the docker image, etc. VirtualBox under minikube tries to do that, but it doesn't work reliably. 

So I was intending to find a nice solution for people who are developing locally with istio+updating the docs; IMHO minikube is great for this. Otherwise, it's up to you to figure out the docker-machine, shared folders, sync the shared folders with the volume mounts in the docker images, etc and that can be a pain. Although, for docker-machine with xhyve i suspect it would work just like it does with minikube though i've not yet tried. Want me to change some of that text to just be "Minikube" specific instead?